### PR TITLE
NXP-26897: build elements from monorepo

### PR DESCRIPTION
**Pull request by Gabez0r**

### CODE REVIEW

File: Jenkinsfiles/web-ui-pipeline.groovy [hunk 1]
```diff
@@ -98,13 +98,12 @@ timestamps {
             }
             stage('nuxeo-ui-elements') {
                 timeout(30) {
-                    uiel = cloneRebaseAndDir('nuxeo-ui-elements', BRANCH, ELEMENTS_BASE_BRANCH)
-                    if (uiel || el) {
+                    if (elrepo) {
                         echo 'Need to build nuxeo-ui-elements'
-                        dir('nuxeo-ui-elements') {
+                        dir('nuxeo-elementss/ui') {
```
Comment (nmpcunha) [inline Jenkinsfiles/web-ui-pipeline.groovy]: An extra `s`?

File: Jenkinsfiles/web-ui-pipeline.groovy [hunk 2]
```diff
@@ -77,16 +77,16 @@ timestamps {
             def VERSIONS_MAPPING = ['10.10': '2.4', '9.10': '2.2']
             def ELEMENTS_BASE_BRANCH = VERSIONS_MAPPING.containsKey(BASE_BRANCH) ? "maintenance-${VERSIONS_MAPPING.get(BASE_BRANCH)}.x" : BASE_BRANCH
             def MP_BASE_BRANCH = VERSIONS_MAPPING.containsKey(BASE_BRANCH) ? "${VERSIONS_MAPPING.get(BASE_BRANCH)}_${BASE_BRANCH}" : BASE_BRANCH
-            def el, uiel, webui, webuiitests, plugin
+            def elrepo, el, uiel, datavizel, webui, webuiitests, plugin
             if (params.CLEAN) {
                 sh 'npm cache clean --force'
             }
+            elrepo = cloneRebaseAndDir('nuxeo-elements', BRANCH, ELEMENTS_BASE_BRANCH)
```
Comment (nelsonsilva) [inline Jenkinsfiles/web-ui-pipeline.groovy]: Can't we just do a single if(cloneRebaseAndDir) ... ?

Comment (Gabez0r) [reply Jenkinsfiles/web-ui-pipeline.groovy]: We can. I was using `elrepo` later on, but am not any more. Thanks.

**Comment by guirenard** [comment]
:+1:

**Comment by nelsonsilva** [comment]
👍

**Pull request merged**